csrc/preseg_passes/add_axioms.h

```diff
@@ -16,6 +16,9 @@ class AddAxiomsPass : public OptimizationPass<AddAxiomsPass> {
 
  protected:
   NVF_API static void runPass(Fusion* fusion);
+  NVF_API static std::string name() {
```

Comment:
I understood you were copying the NVF_API from the other method. I doubt these NVF_APIs are needed, so let's try to remove them and make sure CI passes.

Reply:
You are right. It's a copy-paste. I'll remove & check CI.